Description: Increment PREREQ needed for gnulib
Author: Alastair McKinstry <mckinstry@debian.org>
Bug-Origin: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=948511
Last-Updated: 2020-01-22
Forwarded: no

Index: libtool/configure.ac
===================================================================
--- libtool.orig/configure.ac
+++ libtool/configure.ac
@@ -24,7 +24,7 @@
 ####
 
 
-AC_PREREQ(2.62)
+AC_PREREQ(2.63)
 dnl Oldest automake required for bootstrap is below in AM_INIT_AUTOMAKE.
 
 
