Description: Updated dependency needed for Gnulib
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=979173
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2021-01-04
Forwarded: no

Index: libtool-2.4.6/configure.ac
===================================================================
--- libtool-2.4.6.orig/configure.ac
+++ libtool-2.4.6/configure.ac
@@ -24,7 +24,7 @@
 ####
 
 
-AC_PREREQ(2.63)
+AC_PREREQ(2.64)
 dnl Oldest automake required for bootstrap is below in AM_INIT_AUTOMAKE.
 
 
