--- a/tests/deplibs-ident.at
+++ b/tests/deplibs-ident.at
@@ -67,13 +67,6 @@
   AT_CHECK([$LIBTOOL --mode=link $CC $CFLAGS $LDFLAGS -o b$EXEEXT b.$OBJEXT ../liba1.la ../liba2.la ../liba3.la ../../c/libcee.la -rpath /nowhere],
 	   [0],[stdout],[ignore])
   AT_CHECK([$EGREP 'cee.*cee' stdout], 1, [ignore], [ignore])
-  AT_XFAIL_IF([dnl
-    eval `$LIBTOOL --config | $EGREP '^hardcode_(direct|direct_absolute|action)='`
-    case $hardcode_action,$hardcode_direct,$hardcode_direct_absolute in
-      relink,yes,no) :;;
-      *,no,*) :;;
-      *) false;;
-    esac])
   dnl This is currently broken in libtool
 )
 
