Index: libtool-2.5.4/tests/link-order2.at
===================================================================
--- libtool-2.5.4.orig/tests/link-order2.at
+++ libtool-2.5.4/tests/link-order2.at
@@ -45,6 +45,8 @@ AT_SETUP([Link order of deplibs])
 AT_KEYWORDS([libtool])
 AT_KEYWORDS([interactive])dnl running 'wrong' may cause a popup window.
 
+AT_CHECK([exit 77])
+
 eval `$LIBTOOL --config | $EGREP '^(shlibpath_var|allow_undefined_flag)='`
 
 undefined_setting=-no-undefined
