Index: libtool-2.5.4/m4/libtool.m4
===================================================================
--- libtool-2.5.4.orig/m4/libtool.m4
+++ libtool-2.5.4/m4/libtool.m4
@@ -2978,6 +2978,18 @@ netbsdelf*-gnu)
   dynamic_linker='NetBSD ld.elf_so'
   ;;
 
+netbsdelf*-gnu)
+  version_type=linux
+  need_lib_prefix=no
+  need_version=no
+  library_names_spec='${libname}${release}${shared_ext}$versuffix ${libname}${release}${shared_ext}$major ${libname}${shared_ext}'
+  soname_spec='${libname}${release}${shared_ext}$major'
+  shlibpath_var=LD_LIBRARY_PATH
+  shlibpath_overrides_runpath=no
+  hardcode_into_libs=yes
+  dynamic_linker='NetBSD ld.elf_so'
+  ;;
+
 netbsd*)
   version_type=sunos
   need_lib_prefix=no
