Index: libtool-2.5.4/tests/demo.at
===================================================================
--- libtool-2.5.4.orig/tests/demo.at
+++ libtool-2.5.4/tests/demo.at
@@ -518,7 +518,7 @@ AT_CLEANUP
 AT_SETUP([force non-PIC objects])
 
 AT_CHECK([case $host in
-hppa*|amd64*|x86_64*|s390*)
+hppa*|amd64*|x86_64*|s390*|mips*|arm*|sparc64*)
   # These hosts cannot use non-PIC shared libs
   exit 77 ;;
 *-solaris*|*-sunos*)
